polkadot/node/core/prospective-parachains/src/fragment_chain/mod.rs

```diff
@@ -976,9 +976,14 @@ impl FragmentChain {
 				Ok(()) => {
 					let _ = self.unconnected.add_candidate_entry(candidate);
 				},
-				// Swallow these errors as they can legitimately happen when pruning stale
-				// candidates.
-				Err(_) => {},
+				Err(e) => {
+					gum::trace!(
```

Comment:
Feels wrong to map an `Err` => `trace`, to me `debug` level feels more natural. But maybe it happens very very often..?

Reply:
ERROR log level is used for anything that impacts the functionality of the node or otherwise it is very impotant to make it visible and take some action. This error in the code here does not meet these requirements.